happytransformer/happy_bert.py

```diff
@@ -131,63 +136,72 @@ def __is_one_sentence(self, text):
                     sentence_found = True
                     break
         return True
+#-------------------------------------------------------#
+
+                # QUESTION ANSWERING #
+#-------------------------------------------------------#
+
+    def init_qa(self, model='bert-large-uncased-whole-word-masking-finetuned-squad'):
+        """
+        Initializes the BertForQuestionAnswering transformer
+        NOTE: This uses the bert-large-uncased-whole-word-masking-finetuned-squad pretraining for best results.
+        """
+        self.__qa_model = BertForQuestionAnswering.from_pretrained(model)
+        self.__qa_tokenizer = BertTokenizerFast.from_pretrained(model)
+        self.__qa_model.eval()
+
+        if self.gpu_support == 'cuda':
+            self.__qa_model.to('cuda')
+
+        self.__qa_runner = AnswerQuestionRunner(self._model_name, self.__qa_model, self.__qa_tokenizer)
+        self.__qa_init = True
+
+    def answers_to_question(self, question, context, k=3):
+        if self.__qa_init:
```

Comment:
It seems weird that MLM and NSP models are loaded by default, but QA models need to have an initialization method

Reply:
In version 2.0, every task will follow the structure proposed in this PR. This will result in breaking changes, but it's worth it in the long run.